content/06.methods.md

```diff
@@ -570,8 +565,14 @@ We predicted telomerase activity of tumor samples using the recently developed E
 In brief, EXTEND estimates telomerase activity based on the expression of a 13-gene signature. 
 We derived this signature by comparing telomerase-positive tumors and tumors with activated alternative lengthening of telomeres pathway, a group presumably negative of telomerase activity. 
 
-### ADDITIONAL RESOURCES
-<!-- TODO: Add section -->
+### Survival models
+
+We calculated overall survival (OS), as days since initial diagnosis (`OS_days`) and progression-free survival (PFS), as days since last event (`PFS_days`).
+Using the [`survival` R package](https://github.com/therneau/survival), we performed survival analysis for patients by HGG subtype using the Kaplan-Meier estimator [@doi:10.2307/2281868] and a log-rank test (Mantel-Cox test) [@pmid:5910392] on the different HGG subtypes.
+Next, we used multivariate cox (proportional hazards) regression analysis [@url:http://www.jstor.org/stable/2985181] to model the following:
+- `tp53 scores + telomerase scores + extent of tumor resection + LGG group + HGG group`, in which `tp53 scores` and `telomerase scores` are numeric, `extent of tumor resection` is categorical, and `LGG group` and `HGG group` are binary variables indicating group status.
+- `tp53 scores + telomerase scores + extent of tumor resection` for each `cancer_group` with an N>=3 deceased patients (DIPG, DMG, HGG, MB, and EPN)
+- `quantiseq cell type fractions + CD274 expression + extent of tumor resection` for each `cancer_group` with an N>=3 deceased patients (DIPG, DMG, HGG, MB, and EPN), in which `quantiseq cell type fractions` and `CD274 expression` are numeric.
```

Comment:
These aren't coming up as bullets. This could either be a spacing thing, or maybe manubot just doesn't like bullets?

If the problem is manubot not liking bullets, I think the next best thing is to format this way - 
"...to model the following: a) first formula where blah blah blah; b) second formula where blah blah blah"

Reply:
sure, will do that